src/Contract/Wallet/KeyFile.purs

```diff
@@ -29,8 +37,21 @@ import Node.Path (FilePath)
 -- |
 -- | **NodeJS only**
 mkKeyWalletFromFiles
-  :: FilePath -> Maybe FilePath -> Aff Wallet.Wallet
+  :: FilePath -> Maybe FilePath -> Contract KeyWallet
```

Comment:
This function should be turned back to the `Aff` type. The below one can be removed.

It was in `Aff` because a wallet is needed to initialize the `Contract`, so users are expected to call it before `Contract`.

Reply:
> a wallet is needed to initialize the `Contract`

Not really true, you can have no wallet in a contract and use `withKeyWallet` once the wallet is constructed, although perhaps not obvious to users

This change occurred after I made `KeyWallet` require a `NetworkId` upon construction. This made code elsewhere more streamlined, but also conceptually, why would we need a key wallet to be polymorphic to network after construction?

Keeping it in `Contract` helps avoid issues with wrong `NetworkId`s

Reply:
> why would we need a key wallet to be polymorphic to network after construction?

Because it's just a wrapped Ed25519 private key . What exactly do we need the network ID for? If it's just to tag it, I'm strongly against this change.

> Not really true, you can have no wallet in a contract and use withKeyWallet once the wallet is constructed

OK, but then there's another argument: we should not ask the user to provide more arguments than we need to perform the computation. `Contract` environment is not only the network ID parameter, but also the runtime handles, that we should not use. We don't want to force the users to initialize it just to get a KeyWallet (e.g. for later use in Contract, or just to call `signData`).

Reply:
Okay, I was thinking `KeyWallet` existed mostly for consumption in `Contract`, so it made sense to make it similar to other wallets (which are also bound to a network).

> If it's just to tag it, I'm strongly against this change.

It's not just as a tag, it also removes the need to pass in the network id every time we want the key wallets `address` or to `signData`.

I thought the `Aff` version would be sufficient to handle the non-contract cases, but I'm happy to revert the change

Reply:
Reverted